src/megachatapi_impl.cpp

```diff
@@ -4087,6 +4087,70 @@ int MegaChatApiImpl::getMaxVideoCallParticipants()
     return rtcModule::IRtcModule::kMaxCallVideoSenders;
 }
 
+bool MegaChatApiImpl::isAudioMonitorEnabled(MegaChatHandle chatid)
+{
+    SdkMutexGuard g(sdkMutex);
+    if (!mClient->rtc)
+    {
+        API_LOG_ERROR("isAudioMonitorEnabled - WebRTC is not initialized");
+        return false;
+    }
+
+    if (chatid != MEGACHAT_INVALID_HANDLE)
+    {
+        API_LOG_ERROR("isAudioMonitorEnabled - Invalid chatid");
+        return false;
+    }
+
+    MegaChatCallHandler *handler = findChatCallHandler(chatid);
+    if (!handler)
+    {
+        API_LOG_ERROR("isAudioMonitorEnabled - Failed to get the call handler associated to chat room");
+        return false;
+    }
+
+    rtcModule::ICall *call = handler->getCall();
+    if (!call)
+    {
+        API_LOG_ERROR("isAudioMonitorEnabled - There is not any call associated to MegaChatCallHandler");
+        return false;
+    }
+
+    return call->isAudioMonitorEnabled();
+}
+
+void MegaChatApiImpl::enableAudioMonitor(bool enable, MegaChatHandle chatid)
```

Comment:
Maybe this method should return the effective value. Otherwise, callers will need to check, with an additional call to `isAudioMonitorEnabled()`, if the call was successful or failed.

Reply:
Fixed